.github/workflows/autoreplies/check_account_recovery.py

```diff
@@ -0,0 +1,187 @@
+    github_issue_owner = os.environ.get("GITHUB_ISSUE_OWNER", "pypi")
+    github_issue_repo = os.environ.get("GITHUB_ISSUE_REPO", "support")
+
+    issue_data = gh_utils.fetch_issue_details(
+        github_issue_owner, github_issue_repo, issue_number, github_token=github_token
+    )
+
+    gh_user = issue_data["user"]
+    gh_user_link = format_markdown_gh_user_link(gh_user)
+
+    if PYPI_USER_HEADER not in issue_data["body"]:
+        raise ValueError(f"Issue body does not contain expected header: {PYPI_USER_HEADER}")
+
+    pypi_user = sanitize_pypi_user(issue_data["body"]["PyPI Username"])
+    pypi_user_link = format_markdown_pypi_user_link(pypi_user)
+
+    try:
+        packages = pypi_utils.get_packages_by_user(pypi_user)
+    except ValueError as e:
+        raise e
+
+    # If the pypi user is not a maintainer for any packages
+    if not packages:
+        gh_utils.add_issue_comment(
+            f"User {pypi_user_link} has no packages",
+            github_issue_owner,
+            github_issue_repo,
+            issue_number,
+            github_token=github_token,
+        )
+        sys.exit()
+
+    # Loop over all packages to see if they belong to the user
+    package_ownership = []  # List of [package_name, repo_url, ownership_status]
+    for package_name in packages:
+        pypi_package_link = format_markdown_package_link(package_name)
+        package = pypi_utils.get_pypi_project_info(package_name)
+
+        # Package has source code repo listed at PyPI
+        if "repository_url" not in package or not package["repository_url"]:
+            package_ownership.append([pypi_package_link, "", NO_REPO])
+            continue
+
+        package_repo_url = package["repository_url"]
+
+        # Package source repo directly belongs to the gh_user
+        if gh_utils.does_user_own_repo(package_repo_url, gh_user):
+            package_ownership.append([pypi_package_link, package_repo_url, BELONGS])
+            continue
+
+        # If package source repo belongs to an organization - check if the gh_user is a member or admin
+        org_status = gh_utils.get_user_role_in_org(package_repo_url, gh_user)
+        if org_status == "admin":
+            package_ownership.append([pypi_package_link, package_repo_url, ORG_ADMIN])
+        elif org_status == "member":
+            package_ownership.append([pypi_package_link, package_repo_url, ORG_MEMBER])
+
+        # Otherwise the source repo may not belong to the gh_user
+        else:
+            package_ownership.append([pypi_package_link, package_repo_url, UNKNOWN_OWERNSHIP])
+
+    # Add a comment to the issue with the package ownership information
+    table = format_markdown_table(package_ownership)
+
+    # Count how many packages are not owned or administered by the user
+    num_unverified = len([row for row in package_ownership if row[2] > ORG_ADMIN])
+
+    if num_unverified == 0:
+        label = "fasttrack"
+    else:
+        label = ""
```

Comment:
I don't know what your policy would call for here. This code considers repos to be owned by the user if they are directly owned, or if they belong to an organization that the github user is an admin for.